packer/scylla_install_image

```diff
@@ -112,6 +113,16 @@ if __name__ == '__main__':
         run('dpkg -i /tmp/amazon-ssm-agent.deb', shell=True, check=True)
         run('systemctl enable amazon-ssm-agent', shell=True, check=True)
 
+        run('apt-get install -y linux-aws-lts-22.04', shell=True, check=True)
+        kernel_version = subprocess.check_output("apt-cache depends linux-image-aws-lts-22.04 | grep Depends: |awk -F'linux-image-' '{print $2}' | head -n1", shell=True).decode('utf-8').strip()
+        mid = subprocess.check_output("awk '/Advanced options for Ubuntu/{print $(NF-1)}' /boot/grub/grub.cfg", shell=True).decode().strip().split("'")[1]
+        kid = subprocess.check_output(f"awk '/with Linux {kernel_version}/{{print $(NF-1)}}' /boot/grub/grub.cfg | head -n1", shell=True).decode().strip().split("'")[1]
```

Comment:
use `text=True`, instead of `decode()`
```suggestion
        kernel_version = subprocess.check_output("apt-cache depends linux-image-aws-lts-22.04 | grep Depends: |awk -F'linux-image-' '{print $2}' | head -n1", shell=True, text=True).strip()
        mid = subprocess.check_output("awk '/Advanced options for Ubuntu/{print $(NF-1)}' /boot/grub/grub.cfg", shell=True, text=True).strip().split("'")[1]
        kid = subprocess.check_output(f"awk '/with Linux {kernel_version}/{{print $(NF-1)}}' /boot/grub/grub.cfg | head -n1", shell=True, text=True).strip().split("'")[1]
```